app/src/main/java/com/app/worte/worte/FileSystemOperator.java

```diff
@@ -37,14 +44,16 @@ private void parseDbFile(File dbFile)
             String line = reader.readLine();
             while (line != null)
             {
-                String[] splited = line.split(" - ");
+                String[] splited = line.split(" \\| ");
 
-                if(splited.length == 2)
+                if(splited.length == 3)
                 {
                     String newLanguage = splited[0];
-                    String nativeLanguage = splited[1];
+                    Log.w(LOG_TAG, "newLanguage: " + newLanguage);
+                    String origLanguage = splited[1];
+                    int knowledge = Integer.parseInt(splited[2]);
```

Comment:
Can `parseInt` fail? What will happen in case of its failure?

Reply:
added exception handling